Packages/link.magic.unity.sdk/Runtime/Magic.cs

```diff
@@ -14,22 +15,22 @@ public sealed class Magic
         public readonly UserModule User;
 
         //Constructor
-        public Magic(string apikey, EthNetwork network = EthNetwork.Mainnet, string locale = "en-US")
+        public Magic(string apikey, EthNetwork network = EthNetwork.Mainnet, string locale = "en-US", GameObject canvas = null)
```

Comment:
What do you think giving canvas a non-generic name, as it's only needed for Editor and Mac Support. Something like MacEditorCanvas